docs/source/user-guide/integrations/hugging_face.rst

```diff
@@ -0,0 +1,233 @@
+HuggingFace Transformers
+========================
+
+This tutorial demonstrates how deepchecks.vision can be used on a Hugging Face transformer model. We will use deepchecks
+to compare the performance of the `DETR ResNet <https://huggingface.co/facebook/detr-resnet-50>`__ transformers model
+against the widely used `YOLOv5s <https://arxiv.org/abs/1804.02767>`__ model on the `COCO <https://cocodataset.org/>`__
+dataset.
+
+Load COCO and YOLOv5s
+------------------------
+
+We'll start by loading from ``deepchecks.vision.datasets.detection.coco`` a sample of the COCO dataset (coco 128) and
+the YOLO model, both downloaded from `ultralytics <https://github.com/ultralytics/yolov5>`__ repository.
+
+.. code-block:: python
+
+    from deepchecks.vision.datasets.detection import coco
+    yolo_train_ds = coco.load_dataset(object_type='VisionData')
+    yolo_test_ds = coco.load_dataset(object_type='VisionData', train=False)
+    yolo = coco.load_model()
+
+We already loaded the data wrapped with the relevant ``VisionData`` object, so we can just use the
+:doc:`MeanAveragePrecisionReport </checks_gallery/vision/model_evaluation/plot_mean_average_precision_report>` check to
+evaluate the model's performance for various IoU thresholds and bounding box sizes.
+
+.. code-block:: python
+
+    from deepchecks.vision.checks import MeanAveragePrecisionReport
+
+    device = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
+    yolo_map_result = MeanAveragePrecisionReport().run(yolo_test_ds, yolo, device=device)
+    yolo_map_result.show()
+
+.. image:: /_static/yolo_map.png
+   :alt: Mean Average Precision Report for Yolov5
+   :align: center
+
+
+Implement a DetectionData Class for the DETR Model
```

Comment:
consider adding here something to the title to make it clear that this is the "adapting to hugging face transformer part" - (necessarily especially if we leave the part before there. if not then that does somewhat solve it because this will be first and clearer that this is the "content part")